crates/flipperzero/examples/gui.rs

```diff
@@ -28,7 +28,7 @@ manifest!(name = "Rust GUI example");
 entry!(main);
 
 /// View draw handler.
-pub extern "C" fn draw_callback(canvas: *mut sys::Canvas, _context: *mut c_void) {
+pub unsafe extern "C" fn draw_callback(canvas: *mut sys::Canvas, _context: *mut c_void) {
```

Comment:
Was this change intended?

Reply:
Yup, it's due to the fact that while function is `extern "C"`, it still is normally callable from safe thus not having `unsafe` on it (considering the fact that it's dereferencing a raw pointer) is unsound.****